qcengine/procedures/optking.py

```diff
@@ -0,0 +1,48 @@
+from typing import Any, Dict, Union
+
+from qcelemental.models import OptimizationResult, OptimizationInput
+from qcelemental.util import which_import
+
+from .model import ProcedureHarness
+
+
+class OptKingProcedure(ProcedureHarness):
+
+    _defaults = {"name": "OptKing", "procedure": "optimization"}
+
+    class Config(ProcedureHarness.Config):
+        pass
+
+    def found(self, raise_error: bool = False) -> bool:
+        return which_import(
+            "optking",
+            return_bool=True,
+            raise_error=raise_error,
+            raise_msg="Please install via `conda install optking -c psi4`.",
```

Comment:
`-c psi4/label/dev` for the immediate future, not that I can get it uploaded to either.